CHANGELOG.md

```diff
@@ -5,10 +5,30 @@ next release
 -------------------
 ### Backend Changes
 #### Breaking Changes
+
+#### New Features
+
+1.29.0 (2021-12-01)
+-------------------
+### Backend Changes
+
+#### Breaking Changes
+
 * Remove deprecated `--badger.truncate` CLI flag in [#3410](https://github.com/jaegertracing/jaeger/pull/3410))
 
 #### New Features
 
+* Expose rackID option in ingester [#3395](https://github.com/jaegertracing/jaeger/pull/3395))
```

Comment:
the format of items does not follow the previous format (author, PR).  Also the breaking change item on line 17 should be changed

Reply:
PR updated, but not sure what you meant regarding line 17. Does it look OK now, or should it still be changed?

Reply:
it looks fine now. It didn't look good previously so I explicitly mentioned it as it was not added in this PR